frontend/src/components/personal/PreferencesPage.tsx

```diff
@@ -39,7 +40,7 @@ export default function PreferencesPage (): React.ReactElement {
     setEmail(e.target.value)
   }, [setEmail])
 
-  const [id, setId] = useState(1)
+  const [id, setId] = useState(user? user.id : 1)
```

Comment:
I think we should not use default value here

Reply:
corrected as required